docs/configuration.md

```diff
@@ -702,15 +702,15 @@ Obviously, you can't set repository or package file location with this method.
 </tr>
 <tr>
   <td>`assignees`</td>
-  <td>Assignees for Pull Request</td>
+  <td>Assignees for Pull Request (with email in vsts, user name in GitHub)</td>
```

Comment:
This table should not be edited directly, as it gets regenerated regularly and hence would be overwritten later. Instead, please edit `lib/config/definitions.js` and run `npm run update-docs`

Reply:
Hooo, sorry. I got it. I'll update this tonight.

Reply:
Done 👍